tests/lib/reset.sh

```diff
@@ -2,18 +2,8 @@
 
 set -e -x
 
-systemctl stop snapd || true
-mounts="$(systemctl list-unit-files | grep '^snap[-.].*\.mount' | cut -f1 -d ' ')"
-services="$(systemctl list-unit-files | grep '^snap[-.].*\.service' | cut -f1 -d ' ')"
-for unit in $services $mounts; do
-    systemctl stop $unit || true
-done
-
-rm -f /tmp/ubuntu-core*
-rm -rf /snap/*
-rm -rf /var/lib/snapd/*
-rm -f /etc/systemd/system/snap[-.]*.{mount,service}
-rm -f /etc/systemd/system/multi-user.target.wants/snap[-.]*.{mount,service}
+# purge all state
+$PROJECT_PATH/debian/snapd.postrm purge
```

Comment:
I think the style is to use SPREAD_PATH here (PROJECT_PATH is a spread.yaml thing)